src/Symfony/Component/Ldap/Security/LdapUser.php

```diff
@@ -62,6 +62,8 @@ public function getUserIdentifier(): string
 
     public function eraseCredentials(): void
     {
+        trigger_deprecation('symfony/security-core', '7.3', sprintf('The "%s()" method is deprecated and will be removed in 8.0, use a DTO instead or implement your own erasing logic if needed.', __METHOD__));
```

Comment:
Where do we set the password to null now? It cannot/should not be the responsibility of the developer as this is a built-in class, right?

Reply:
Good point, added an `EraseLdapUserCredentialsListener` which does it instead.